### PR TITLE
Mobile js

### PR DESCRIPTION
**Pull request by vanina123**
- add popup menu.
- add image.
- modify mobile.js

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -116,6 +124,395 @@ li {
...
+  margin-top: 30px;
+  padding: 20px;
+  gap: 12px;
+  background: #fff;
+  border-radius: 16px;
+  -webkit-backdrop-filter: blur(8px);
+  backdrop-filter: blur(8px);
+}
+
+.portfolio-modal-img img {
```
Comment (BregornOriginal) [inline styles.css:177]: - Check the position of the elements, some of them are outside of the screen. To be responsive they should be placed inside the card while you change the size of the window.

image

File: styles.css [hunk 2]
```diff
@@ -116,6 +124,395 @@ li {
...
+  margin-right: auto;
+  margin-top: 30px;
+  padding: 20px;
+  gap: 12px;
+  background: #fff;
+  border-radius: 16px;
+  -webkit-backdrop-filter: blur(8px);
+  backdrop-filter: blur(8px);
+}
+
```
Comment (BregornOriginal) [inline styles.css:176]: - Kindly align the grey line to the center as it is on the Figma design

image

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
## Hi @vanina123 

## Good job so far!

## There are some issues that you still need to work on to go to the next project but you are almost there!

one-step-closer-one-more

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good pull request description

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @vanina123 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done on making the required changes from the previous reviewer

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**